docs/fastdds/dds_layer/publisher/publisher/publisher.rst

```diff
@@ -3,3 +3,160 @@
+
++--------------------------------+------------------------------------+----------+
+| QosPolicy class                | Accessor                           | Mutable  |
++================================+====================================+==========+
+| PresentationQosPolicy          | :func:`presentation`               | Yes      |
++--------------------------------+------------------------------------+----------+
+| PartitionQosPolicy             | :func:`partition`                  | Yes      |
++--------------------------------+------------------------------------+----------+
+| GroupDataQosPolicy             | :func:`group_data`                 | Yes      |
++--------------------------------+------------------------------------+----------+
+| EntityFactoryQosPolicy         | :func:`entity_factory`             | Yes      |
++--------------------------------+------------------------------------+----------+
+
+Refer to the detailed description of each :class:`QosPolicy` class for more information about their usage and
+default values.
+
+The QoS value of a previously created :ref:`dds_layer_publisher_publisher` can be modified using the
+:func:`set_qos` member function.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_PUBLISHERQOS
+   :end-before: //!
+
+
+.. _dds_layer_defaultPublisherQos:
+
+Default PublisherQos
+^^^^^^^^^^^^^^^^^^^^
+
+The default :ref:`dds_layer_publisher_publisherQos` refers to the value returned by the
+:func:`get_default_publisher_qos` member function on the :ref:`dds_layer_domainParticipant` instance.
+The special value ``PUBLISHER_QOS_DEFAULT`` can be used as QoS argument on :func:`create_publisher`
+or :func:`set_qos` member functions to indicate that the current default :ref:`dds_layer_publisher_publisherQos`
+should be used.
+
+When the system starts, the default :ref:`dds_layer_publisher_publisherQos` is equivalent to the default constructed
+value :func:`PublisherQos`.
+The default :ref:`dds_layer_publisher_publisherQos` can be modified at any time using the
+:func:`set_default_publisher_qos` member function on the :ref:`dds_layer_domainParticipant` instance.
+Modifying the default :ref:`dds_layer_publisher_publisherQos` will not affect already existing
+:ref:`dds_layer_publisher_publisher` instances.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_DEFAULT_PUBLISHERQOS
+   :end-before: //!
+
+:func:`set_default_publisher_qos` member function also accepts the special value ``PUBLISHER_QOS_DEFAULT``
+as input argument.
+This will reset the current default :ref:`dds_layer_publisher_publisherQos` to default constructed
+value :func:`PublisherQos`.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_PUBLISHERQOS_TO_DEFAULT
+   :end-before: //!
+
+
+.. _dds_layer_publisher_creation:
+
+Creating a Publisher
```

Comment:
I'd put this and `Creating a DataWriter` as standalone pages. Same spot in the toctree and all, but just by themselves